### ISSUE TITLE
Cancel download using Ctrl+C/SIGINT

### ISSUE THREAD

**Issue by clemens-schulz** [jiaweihli/manga_downloader#69]
There is no option to cancel downloads. You can cancel downloads using Ctrl+Z & kill, but this won't remove temp files.

**Comment by jiaweihli** [comment]
Yep, this could definitely be improved - need to add a hook to listen for the interrupt and kill all threads.  I'm caught up in another side project at the moment, but would happily accept a PR!
